apps/src/lib/ui/lti/link/LtiLinkAccountPage/WelcomeBanner/index.tsx

```diff
@@ -1,54 +1,37 @@
 import styles from '@cdo/apps/lib/ui/lti/link/LtiLinkAccountPage/link-account.module.scss';
-import FontAwesomeV6Icon from '@cdo/apps/componentLibrary/fontAwesomeV6Icon';
 import Typography from '@cdo/apps/componentLibrary/typography/Typography';
-import React, {useCallback, useContext} from 'react';
+import React, {useContext} from 'react';
 import i18n from '@cdo/locale';
 
-import canvas from '../../../assets/canvas.svg';
-import schoology from '../../../assets/schoology.svg';
 import codeLogo from '../../../assets/codeLogo.svg';
 import {LtiProviderContext} from '../context';
 
 const WelcomeBanner = () => {
-  const {ltiProvider, ltiProviderName} = useContext(LtiProviderContext)!;
-
-  const getLtiProviderIcon = useCallback(() => {
-    switch (ltiProvider) {
-      case 'canvas_cloud':
-      case 'canvas_beta_cloud':
-      case 'canvas_test_cloud':
-        return canvas;
-      case 'schoology':
-        return schoology;
-      default:
-        return undefined;
-    }
-  }, [ltiProvider]);
-
-  const ltiProviderIcon = getLtiProviderIcon();
+  const {ltiProviderName} = useContext(LtiProviderContext)!;
+  const isLMS = true;
```

Comment:
This is for later, not necessarily as part of this PR, but I wanted to check your thoughts on when you circle back to refactor LMS vs non-LMS, I think we can use some compositional branching logic for all of the components in this directory (perhaps generalizing this page completely), I'm thinking of something like:

`WelcomeBanner` switches to `LtiWelcomeBanner` or `ProfessionalDevelopmentWelcomeBanner` depending on the type.

I'd like to avoid having a single WelcomeBanner component that encapsulates all the different types. (This comment applies to all the components)